pkgs/by-name/ta/talculate/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  fetchPypi,
+}:
+
+let
+  textual = python3.pkgs.textual.overridePythonAttrs (old: {
+    version = "3.7.0";
+    src = fetchPypi {
+      pname = "textual";
+      version = "3.7.0";
+      sha256 = "sha256-1ctxC0aRotm5B1/y3J6+zy2DaciTrzNU2tNElgyR18k=";
+    };
+    doCheck = false;
+  });
+
+  tree-sitter = python3.pkgs.tree-sitter.overridePythonAttrs (old: {
+    version = "0.21.0";
+    src = fetchPypi {
+      pname = "tree-sitter";
+      version = "0.21.0";
+      sha256 = "sha256-x07J7/MODFufAO5XjMpk3zIrmIXIoVNkosU39IWrzHc=";
+    };
+    doCheck = false;
+  });
+in
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "talculate";
+  version = "unstable-2025-07-25";
```

```suggestion
  version = "0-unstable-2025-07-25";
```
Versions must start with a digit, and, if there's no tagging or versioning whatsoever, the format is `0-unstable-DATE`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning